src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -0,0 +1,27 @@
+package juja.sqlcmd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DatabaseManagerTest {
+    DatabaseManager databaseManager;
+
+    @Before
+    public void init() {
+        databaseManager = new DatabaseManager();
+    }
+
+    @Test
+    public void testSuccessfulConnection() {
+        boolean expected = databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd");
+        assertEquals(expected, true);
+    }
+
+    @Test
+    public void testUnSuccessfulConnection() {
```

Comment:
I think 'Unsuccessful' is particular word. Decouple on two words is unnecessary maybe.

Reply:
Sure, thanks... And testFailedConnection() even beter name, I think. Will rename this test